src/index.js

```diff
@@ -25,6 +25,7 @@ program
   )
   .option("--username [username]", "Username for basic auth (Hydra only)")
   .option("--password [password]", "Password for basic auth (Hydra only)")
+  .option("--jwt [bearer]", "JWT auth (Hydra only)")
```

Comment:
```suggestion
  .option("--bearer [bearer]", "Token for bearer auth (Hydra only)")
```

Because this will also work for non-JWT bearer token (e.g. Paseto, or random tokens)